### PR TITLE
CI SetUp All Tests Passing

### PR DESCRIPTION
**Pull request by xrimpy**
Test Setup for CI BuiildKite on Docker-Compose ONly

### CODE REVIEW

**Comment by akhilgkrishnan** [comment]
@xrimpy Still the CI is failing

**Comment by xrimpy** [comment]
Docker-Compose is working on the server when triggered manually. Checking the connection settings.

File: docker-compose.yml [hunk 1]
```diff
@@ -42,21 +42,6 @@ services:
     tmpfs:
       - /tmp/pids/
 
-  webpack:
```
[the remark this answers is not in the document]
Comment (xrimpy) [reply docker-compose.yml]: Restored. Re-running the tests

File: docker-compose.yml [hunk 2]
```diff
@@ -24,38 +24,42 @@ services:
...
-    ports:
-      - "3035:3035"
-    environment:
-      - WEBPACKER_DEV_SERVER_HOST=0.0.0.0
-      - DATABASE_URL=postgres://postgres:postgres@database:5432/
-      - REDIS_URL=redis://redis:6379/1
-      - ELASTICSEARCH_URL=http://elasticsearch:9200
-    depends_on:
-      - database
+  # webpack:
```
Comment (akhilgkrishnan) [inline docker-compose.yml:47]: any reason for commenting this

**Review by akhilgkrishnan — COMMENTED** [review]
@xrimpy added one comment

File: docker-compose.yml (hunk 2 above)
Comment (xrimpy) [reply docker-compose.yml:47]: It is updated by secure URLs
ELASTICSEARCH_URL=http://elastic:x2Sy5FK51zu@elasticsearch:9200

This avoids warnings when running tests on DockerCompaose

**Review by akhilgkrishnan — APPROVED** [review]
LGTM

**Pull request merged**